apps/derive/elpi/dune

```diff
@@ -0,0 +1,18 @@
+(coq.theory
+ (name elpi.apps.derive.elpi)
+ (package coq-elpi)
+ (theories elpi))
+
+(rule
+ (target dummy.v)
+ (deps
+  (glob_files *.elpi))
+ (action
+  (with-stdout-to %{target}
+   (echo ""))))
+
+(install
+ (files
+  (glob_files (*.elpi with_prefix coq/user-contrib/elpi/apps/derive/elpi/)))
+ (section lib_root)
+ (package coq-elpi))
```

Comment:
This setup seems to work pretty well, and takes care of installing `.elpi` files.

The only drawback is that we need to put these files under a different path (here files are found under `elpi.apps.derive.elpi`, while they were under `elpi.apps.derive` before). That's probably not a big deal?